machines/vc.ts

```diff
@@ -214,6 +217,17 @@ export const vcMachine =
             DELETE_VC: {
               target: 'deletingFailedVcs',
             },
+            VERIFY_VC_FAILED: {
+              actions: [
+                'removeVcFromMyVcs',
+                'removeVcFromInProgressDownlods',
+                'setVerificationErrorMessage',
+              ],
```

Comment:
At Line #59,  we are accepting only errorMessage with VERIFY_VC_FAILED event but here we are removing VC from myVCs why? as verification as failed it means VC was not there in MyVCs list.
In Existing vc machine, we are sending vcMetadata along with event but not from issuer machine why so? We need to have consistency for handling the events.

Reply:
Previously when writing the code we are showing the error message overlay instead of screen and at that time when we click on ok button in the overlay it will remove the error message overlay but in the background the card is stuck in loading state so we had to remove it from myvcs list. Now we are showing screen and when we click on goBack it is going to homescreen and refreshing the vc's so it is not required now we will remove it

Reply:
in existing vc machine if the card is not downloaded after retry limit we are showing download failed error message overlay and removing the Vc from vc.ts myvcs context variable and to remove that we are passing vcMetadata along with the event but in issuer machine we don't have such scenario only if the card is downloaded we will show it and if it is failed for some reason we won't be storing it's metadata and details so no need to pass it.